docs/wiki/feature/atragmx.md

```diff
@@ -14,19 +14,41 @@ version:
 
 ## Overview
 
-### Sub-feature 1
-Short description of sub-feature 1.
+### AtragMx from HORUS VISION
+Horus ATragMX software considers atmospheric conditions, gun data, ammunition, range, speed and muzzle velocity to calculate precise aiming solutions with "come-up" results – and even accounts for Coriolis and spin drift effects. ATragMX, loaded on a handheld computer made by TDS Recon, is easy to use and lightning-fast. The Recon meets the rigorous MIL-STD-810F military standard for drops, vibration, humidity, altitude and extreme temperatures.
 
-### Sub-feature 2
-Short description of sub-feature 2.
+### Official Manuel and Horus Videos
+https://github.com/acemod/ACE3/blob/master/extras/manual_Horus_ATrag-v385.pdf
 
+https://www.youtube.com/watch?v=pg6oqT5jVds
```

Comment:
All the official documentation in the same place seems to me the best solution for a novice user.